### DISCUSSION TITLE
Invitations to #MozSprint (Jun 1-2, esp. because of Privacy Bot)

### DISCUSSION THREAD

**Discussion by Ryuno-Ki** [tosdr/tosdr.org#166] [General]
Dear TOS;DR-team,

I've heard of below invitation and want to forward it to you. Maybe you can share knowledge or even join forces?!

> Cliqz GmbH will be a site host for #MozSprint, if you are around Munich, Germany feel free to join in: https://ti.to/Mozilla/global-sprint-munich. Or, Attend virtually and contribute on some exciting projects. From Cliqz, we have proposed the following privacy centric projects:

> - Green-Analytics: Privacy preserving analytics service. (https://github.com/cliqz-oss/green-analytics)
> - Privacy Bot: Bot which discovers, extracts and save privacy policies off websites. (https://github.com/cliqz-oss/privacy-bot)
> - Web Ninja: An educational game to illustrate issues of tracking and centralisation on the web. (https://github.com/cliqz-oss/web-ninja)

**Comment by ankit-cliqz** [comment]
Thanks @Ryuno-Ki! 

The best project which can fit TOS;DR's scope is the following:
Privacy Bot: Bot which discovers, extracts and save privacy policies off websites. (https://github.com/cliqz-oss/privacy-bot)
Please feel to raise an issue in our repos to reach out to us.
